src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -58,6 +58,56 @@ internal sealed class VirtualProjectBuildingCommand : CommandBase
         "MSBuild.rsp",
     ];
 
+    internal static readonly string TargetOverrides = """
```

Comment:
```suggestion
    internal const string TargetOverrides = """
```

Is there a reason to not make it `const` here?

Reply:
`const` strings are copied to referencing assemblies (so they can blow up their size and potentially make them hit user string literal limit) and since I've learnt that, I use `static readonly` for large `string` constants instead